index.html

```diff
@@ -130,7 +134,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="button">
-                    <button type="button" class="work-button">See Project</button>
+                    <button id="2" type="button" class="work-button">See Project</button>

```

Comment:
- [ ] your popup image is showing half. kindly fix that
![Screenshot (184)](https://user-images.githubusercontent.com/65298048/175521857-2322b04c-f5d2-48b6-b1cb-8f1320f27357.png)